DataLoaderTest.java

```diff
@@ -1,123 +0,0 @@
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("id")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 139);
-            }
-            //10 speciality
-            {
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("speciality")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 10);
-            }
-            //139 mastery
-            {
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("mastery")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 139);
-            }
-            //18 research
-            {
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("research")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 18);
-            }
-            //20 offer
-            {
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("offer")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 20);
-            }
-            //160 name
-            {
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("name")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 160);
-            }
-            //10 school
-            {
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("school")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 10);
-            }
-            //44 year of degree
-            {
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("year_of_degree")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 44);
-            }
-            //11 subject
-            {
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("subject")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 11);
-            }
-            //2 co-teach
-            {
-                TypeQLMatch.Unfiltered.Aggregate query=TypeQL.match(var("x").isa("co-teach")).count();
-                QueryFuture<Numeric> ans=transaction.query().match(query);
-                assertEquals(ans.get().asLong(), 2);
-            }
-        }
-    }
-
-    @After
-    public void disconnect()
-    {
-        session.close();
-        client.close();
-    }
-}
```

Comment:
This little no-entry sign indicates that you're missing a new line at the end of the file. It's convention to have one.